sqlmesh/core/context_diff.py

```diff
@@ -185,6 +185,11 @@ def has_changes(self) -> bool:
     def has_snapshot_changes(self) -> bool:
         return bool(self.added or self.removed or self.modified_snapshots)
 
+    @property
+    def added_internal_models(self) -> t.Set[str]:
```

Comment:
change the name to represent a change to symbolic or materialized

Reply:
https://github.com/TobikoData/sqlmesh/pull/1130/commits/1e8e9505bef66caef0d870612578ea848a844646